src/components/navbar-mobile/index.tsx

```diff
@@ -37,31 +50,48 @@ function NavigationBottom(props: NavigationBottomProps) {
   const isActiveSelect = router.pathname === href
 
   return (
-    <Link href={href}>
+    <RouterLink href={href} passHref>
       <a
         className={clsx(
           'flex justify-center items-center h-full w-7.7 text-xs border-t-4 border-primary-800 dark:border-primary-200 flex-col fill-current',
           !isActiveSelect &&
-            'text-primary-500 dark:text-primary-600 border-t-primary/0 dark:border-t-white/0 fill-transparent',
+            `text-primary-500 dark:text-primary-600 border-t-primary/0 dark:border-t-white/0 fill-transparent`,
```

Comment:
you don't need to use template-strings

```suggestion
            'text-primary-500 dark:text-primary-600 border-t-primary/0 dark:border-t-white/0 fill-transparent',
```